src/main/python/aut/common.py

```diff
@@ -13,3 +13,30 @@ def pages(self):
 
     def links(self):
         return DataFrame(self.loader.extractHyperlinks(self.path), self.sqlContext)
+
+    def images(self):
+        return DataFrame(self.loader.extractImages(self.path), self.sqlContext)
+
+    def image_links(self):
```

Comment:
Sorry for arriving late at the party - some of these DataFrames are singular, e.g., `presentation_program`, while others are plural, e.g., `images`... perhaps we might standardize?

Has this been discussed already?

Also, any issues in calling these what they are? word, powerpoint, excel, etc.?

If so, how about documents, presentations, and spreadsheets?